contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2460,7 +2460,31 @@ SELECT  configuration_id,
         description, 
         is_dynamic, 
         is_advanced 
-FROM sys.babelfish_configurations;
+FROM sys.babelfish_configurations
+UNION ALL
+SELECT 
+CAST(103 as INT) AS configuration_id,
+CAST('user connections' AS SYS.NVARCHAR(35)) AS name,
+CAST(s.setting AS sys.sql_variant) AS value,
+CAST(s.min_val AS sys.sql_variant) AS minimum,
+CAST(s.max_val AS sys.sql_variant) AS maximum,
+CAST(s.setting AS sys.sql_variant) AS value_in_use,
```

Comment:
Actually this is incorrect, s.setting in pg_catalog stores current value and doesnt provide previous value. Need to check how to get the previous value.

Reply:
Why do we need previous value?

Reply:
Because for 'user connections' the config value is not dynamic, it gets applied when the engine is restarted. So value_in_use will be previous value until the engine is restarted and that value will be used by Proxy.

Reply:
Makes sense

Reply:
Instead of adding multiple union all like this, I'll prefer to use as following:
```
SELECT * from babelfish_configurations
union all
SELECT get_config_id(name),
CAST('user connections' AS SYS.NVARCHAR(35)) AS name,
CAST(s.setting AS sys.sql_variant) AS value,
CAST(s.min_val AS sys.sql_variant) AS minimum,
CAST(s.max_val AS sys.sql_variant) AS maximum,
CAST(s.setting AS sys.sql_variant) AS value_in_use,
CAST(s.short_desc AS sys.nvarchar(255)) AS description,
CAST(0 AS sys.BIT) AS is_dynamic,
CAST(1 AS sys.BIT) AS is_advanced
FROM pg_catalog.pg_settings s where name in
('sdsd','sd',...);
```
Also, I think you need to mention collate 'C' while doing the string comparison. Please check with Dipesh.

Reply:
But how to set the configuration_id to 103 for max_connection and 505 for packet size and also the names? we can use if condition but that will have multiple if conditions.